checkstyle.xml

```diff
@@ -14,8 +14,10 @@
 
     Authors: Max Vetrenko, Ruslan Diachenko, Roman Ivanov.
  -->
-
 <module name="Checker">
+    <module name="SuppressionSingleFilter">
+        <property name="files" value="BookMapperImpl.java"/>
+    </module>

```

Comment:
revert it - better add setting to pom xml: specify package that has to be checked by plagion

With the current approach we will need to add all new MapperImpls here